src/domain/session/room/timeline/tiles/BaseMessageTile.js

```diff
@@ -202,4 +215,17 @@ export class BaseMessageTile extends SimpleTile {
             this._reactions.update(annotations, pendingAnnotations);
         }
     }
+
+    get replyTextTile() {
```

Comment:
why text tile rather than tile? is anything limiting this to a `TextTile`?

Reply:
sorry, was only considering text-tiles initially